packages/protocol/migrationsConfig.js

```diff
@@ -202,6 +203,65 @@ const DefaultConfig = {
   },
 }
 
+const NetworkConfigs = {
+  testing: {
+    downtimeSlasher: {
+      slashableDowntime: 6,
+    },
+    election: {
+      minElectableValidators: '1',
+    },
+    epochRewards: {
+      frozen: false,
+    },
+    exchange: {
+      frozen: false,
+    },
+    goldToken: {
+      frozen: false,
+    },
+    stableToken: {
+      frozen: false,
+    },
+    reserve: {
+      initialBalance: 100000000,
+    },
+  },
+  baklava: {
+    blockchainParameters: {
+      minimumClientVersion: {
+        major: 0,
+        minor: 10,
+        patch: 0,
+      },
+    },
+    election: {
+      minElectableValidators: '25', // About half of the expected genesis set.
+    },
+    governance: {
+      // Set to be able to complete a proposal in about a day, but give eveyone a chance to participate.
+      dequeueFrequency: 4 * HOUR,
+      approvalStageDuration: 4 * HOUR,
```

Comment:
Just a heads up that there's a risk that folks submit governance proposals that dequeue and expire before the approvers are able to approve. A 4 hour SLA on governance proposal approval is not something I expect we can offer. Approvers will likely need advance notice of proposals on Baklava as long as this number is so low.

Reply:
At least for the deployment phase, I am intending to make this trade off. We could increase it after the first set of proposals.